### DISCUSSION TITLE
Running Promitor with Docker Compose

### DISCUSSION THREAD

**Discussion by FreddyAyala** [tomkerkhove/promitor#1283] [Q&A] [answered]
Hello there I'm trying to deploy promitor in a simple way using docker-compose, do you have examples of the files az-mon-auth.creds and metric-config.yaml using docker compose?

Thank you very much

**Comment by tomkerkhove** [comment, marked as answer]
I use this configuration locally: https://github.com/tomkerkhove/promitor/tree/master/config/promitor/scraper

az-mon-auth.creds just contains my credentials:
```
PROMITOR_AUTH_APPID=<redacted>
PROMITOR_AUTH_APPKEY=<redacted>
```

This guide should help you get it up and running, if not, all feedback is welcome - https://github.com/tomkerkhove/promitor/blob/master/development-guide.md#running-promitor

**Reply by FreddyAyala** [reply]
Thank you very much, I've deployed correctly the scrapper, for the moment I'm not receiving any metrics, so I wanted to know If i need to also deploy the Service Discovery Container?

**Reply by tomkerkhove** [reply]
It depends what your configuration is. If you are using the configuration that I've liked above in terms of the scraper then it will not work given it's linked to my Azure environment. To emphasize that I've changed the creds sample above.

You will need to configure it to match your landscape.

If you are using Docker Compose, our new resource discovery is included as well which needs to be configured as well.
